README.md

```diff
@@ -1,2 +1,87 @@
-# headless
-TYPO3 JSON API providing content for PWA application
+# TYPO3 Extension "headless"
+Headless allows you to render JSON from TYPO3 content. You can customize output by changing types, names and nesting of fields.
+
+## Requirements
+Extension requires TYPO3 in version at least 9.5.
+
+## Installation
+Install extension using composer\
+``composer require friendsoftypo3/headless``
+
+Then, you should include extension typoscript template, and you are ready to go.
+
+## JSON  Content Object
+In headless extension we implemented new JSON Content Object, which allows you to specify what fields you want to output, and how they will look. First of all, let's take a look at simple example
+```
+lib.page = JSON
+lib.page {
+  fields {
+      header = TEXT
+      header {
+          field = header
+      }
+}
+```
+Output
+```
+{
+    "header" : "headerFieldValue"
+}
+```
+As you can see, we use reserved keyword `fields` to render output. Keys, which are inside `fields` are used as keys in json, so if you specify key `abc = COA`, then in json output you will have
```

Comment:
abc= COA ? I don't get this example. Lets remove it and use the next one only

Reply:
done